docs/user/installation.md

```diff
@@ -6,18 +6,18 @@ There are several ways to install pypdf. The most common option is to use pip.
 
 pypdf requires Python 3.7+ to run.
 
-Typically Python comes with `pip`, a package installer. Using it you can
+Typically Python comes with `pip3`, a package installer. Using it you can
```

Comment:
Makes sense, was unaware of this.
Additionally noticed that change to adding-pdf-annotations.md should be: is, the so-called "Quad points".

Reply:
You can just push further changes to your branch - this will automatically update this PR to fix the remaining issues/review comments. Once this PR is ready to be merged, we squash the commits anyway.